@mattip I have observed the below error while building with `--cpu-dispatch`,
```python

                   from build/src.linux-s390x-3.9/numpy/core/src/umath/loops_comparison.dispatch.vx.c:8:
  numpy/core/src/umath/loops_comparison.dispatch.c.src: In function 'simd_binary_equal_b8':
  numpy/core/src/common/simd/vec/operators.h:148:22: error: '__builtin_s390_vec_eqv' requires z14 or higher
    148 | #define npyv_xnor_b8 vec_eqv
        |                      ^~~~~~~

  numpy/core/src/umath/loops_comparison.dispatch.c.src: In function 'simd_binary_scalar1_less_equal_b8':
  numpy/core/src/common/simd/vec/operators.h:147:21: error: '__builtin_s390_vec_orc' requires z14 or higher
    147 | #define npyv_orc_b8 vec_orc
        |                     ^~~~~~~

```
cc @seiko2plus

--- a/numpy/core/src/common/simd/vec/operators.h
+++ b/numpy/core/src/common/simd/vec/operators.h
@@ -144,12 +144,12 @@ NPY_FINLINE npyv_f64 npyv_not_f64(npyv_f64 a)
 // ANDC, ORC and XNOR
 #define npyv_andc_u8 vec_andc
 #define npyv_andc_b8 vec_andc
-#ifdef NPY_HAVE_VX
-    #define npyv_orc_b8(A, B) npyv_or_b8(npyv_not_b8(B), A)
-    #define npyv_xnor_b8(A, B) npyv_not_b8(npyv_xor_b8(B, A))
-#else
+#if defined(NPY_HAVE_VXE) || defined(NPY_HAVE_VSX)
     #define npyv_orc_b8 vec_orc
     #define npyv_xnor_b8 vec_eqv
+#else
+    #define npyv_orc_b8(A, B) npyv_or_b8(npyv_not_b8(B), A)
+    #define npyv_xnor_b8(A, B) npyv_not_b8(npyv_xor_b8(B, A))
 #endif
 
 /***************************

